ts/screens/profile/components/__test__/ShareDataComponent.test.tsx

```diff
@@ -76,5 +81,11 @@ describe("Test ShareDataComponent", () => {
   });
 });
 
+const globalState = appReducer(undefined, applicationChangeState("active"));
 const renderComponent = () =>
-  render(<ShareDataComponent trackAction={mockTrackInfo} />);
+  renderScreenWithNavigationStoreContext<GlobalState>(
+    () => <ShareDataComponent trackAction={mockTrackInfo} />,
+    "FAKE_ROUTE",
```

Comment:
You can use "DUMMY" as the other component without route in the app 

```suggestion
    "DUMMY",
```